lib/MusicBrainz/Server/Controller/Admin.pm

```diff
@@ -142,7 +131,7 @@ sub delete_user : Path('/admin/user/delete') Args(1) RequireAuth HiddenOnMirrors
 
         $editor = $c->model('Editor')->get_by_id($id);
         $c->response->redirect(
-            $editor ? $c->uri_for_action('/user/profile', [ $editor->name ]) : $c->uri_for('/'));
```

Comment:
This is still needed because users will be removed entirely (no `$editor` left) if they're entirely unreferenced. See https://github.com/metabrainz/musicbrainz-server/blob/0814463ea9d19b4594f92c700c2fa43b899e44db/lib/MusicBrainz/Server/Data/Editor.pm#L618-L633

Reply:
Good catch, reverted in https://github.com/metabrainz/musicbrainz-server/compare/151bbdee5a4f0f740881dad6f34a2d33ce8f5a43..392edbe98c29d5f507c227e7fe40000ccced0fb1